packages/localization/README-ru.md

```diff
@@ -0,0 +1,63 @@
+[English](./README.md)
+
+# Локализация проекта [Quiver UI](https://github.com/tonlabs/UIKit)
+[![PR's приветствуются](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+В этом репозитории размещены файлы локализации проекта Quiver UI. 
```

Comment:
No just Quiver UI, but all the components of UIKit.

Reply:
```suggestion
В этом репозитории размещены файлы локализации UIKit. 
```